corona-warn-app/cwa-app-android
# Bind Dcc wallet info in person details screen (EXPOSUREAPP-11499)

### Testing
- Add any certificate
- Test Menu > CLL Configuration & WalletInfo > Add DCC Wallet Info
- Open person details screen
- The person details screen contains mock data
- The "non-filled shield + checkmark" for VC is removed and not displayed anymore; was previously displayed during the 14-day waiting period (see Jira Story  https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11036)

### Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11499
